functions/exercise21/scriptSpec.js

```diff
@@ -0,0 +1,15 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getPossibleSubset*/
+
+describe("Getting all possible subset with a fixed length combinations in an array", function () {
```

Comment:
Please start using arrow functions in callbacs instead of full anonymous function declaration